Example/iOS/ViewControllers/ControlsDemoViewController.swift

```diff
@@ -0,0 +1,48 @@
+// Created by Cal Stephens on 1/4/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Epoxy
+import EpoxyCollectionView
+import Lottie
+import UIKit
+
+final class ControlsDemoViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    let layout = UICollectionViewCompositionalLayout.list(
+      using: UICollectionLayoutListConfiguration(appearance: .insetGrouped))
+
+    super.init(layout: layout)
+    setItems(items, animated: false)
+
+    title = "Controls Demo"
+  }
+
+  // MARK: Internal
+
+  var items: [ItemModeling] {
```

Comment:
nit: if you'd like you can make this an `@ItemModelBuilder` to omit the commas and array literal